### PR TITLE
Subscriber system testing after merges

### PR DESCRIPTION
**Pull request by salman-codewar**
## Testing criteria

- Add msgClient system in docker-compose
- Test systems integration with new merge changes and msgClient
- Registry, Data-plan and Subscriber systems should work together via docker-compose
- Test Registry sub-systems
- Test Data-Plan sub-systems
- Test Subscriber sub-systems

### CODE REVIEW

File: systems/docker-compose.yml [hunk 1]
```diff
@@ -143,11 +167,14 @@ services:
       - DB_PASSWORD=ThisIsUkamaXPass
       - DB_USER=postgres
       - QUEUE_URI=amqp://guest:guest@rabbitmq:5672
+      - MSGCLIENT_HOST=msg-client-registry:9095
+      - ORG_SERVICE_HOST=org
+      - ORG_SERVICE_PORT=9090
       - UKAMA_UUID=${UKAMA_UUID}
```
Comment (delvatt) [inline systems/docker-compose.yml:173]: I think we dont need that environment variable anymore.

File: systems/docker-compose.yml [hunk 2]
```diff
@@ -159,11 +186,15 @@ services:
       - DB_PASSWORD=ThisIsUkamaXPass
       - DB_USER=postgres
       - QUEUE_URI=amqp://guest:guest@rabbitmq:5672
+      - MSGCLIENT_HOST=msg-client-registry:9095
+      - USERS_SERVICE_HOST=users
+      - USERS_SERVICE_PORT=9090
       - UKAMA_UUID=${UKAMA_UUID}
```
Comment (delvatt) [inline systems/docker-compose.yml:192]: Same as above for UKAMA_UUID

File: systems/docker-compose.yml (hunk 1 above)
Comment (salman-codewar) [reply systems/docker-compose.yml:173]: We need `MSGCLIENT_HOST`, `ORG_SERVICE_HOST ` and `ORG_SERVICE_HOST ` Env's to connecting with msgClient. But yes, you're right about `UKAMA_UUID` i'll remove it.

File: systems/registry/network/cmd/server/main.go [hunk 3]
```diff
@@ -51,7 +51,7 @@ func initConfig() {
 func initDb() sql.Db {
 	log.Infof("Initializing Database")
 	d := sql.NewDb(serviceConfig.DB, serviceConfig.DebugMode)
-	err := d.Init(&db.Org{}, &db.Network{}, &db.Network{})
+	err := d.Init(&db.Org{}, &db.Network{}, &db.Site{})
```
Comment (delvatt) [inline systems/registry/network/cmd/server/main.go:54]: Again, this is weird. That line was already fixed in upstream registry-system base branch.

Comment (salman-codewar) [reply systems/registry/network/cmd/server/main.go:54]: No it get missed somehow, this branch and subscriber-system branch is up-to-date with `main`.

Comment (salman-codewar) [reply systems/registry/network/cmd/server/main.go:54]: Brackley requested me add this line in my PR

**Review by delvatt — APPROVED** [review]
LGTM. With some minor comments.

Also I think in the compose file, even if this is for local testing, still we should be hiding sensitive information like DB usernames, Passwords and root user UUIDs.  So  having maybe  
  ```
      - DEBUGMODE=true
      - DB_HOST=postgresd
      - DB_PASSWORD
      - DB_USER
      - QUEUE_URI=amqp://guest:guest@rabbitmq:5672
      - MSGCLIENT_HOST=msg-client-registry:9095
      - ORG_SERVICE_HOST=org
      - ORG_SERVICE_PORT=9090
      - ORGOWNERUUID
      
```
and passing the effective values through shell environment variables, instead of  hardcoding the values in compose file like the following:
```
      - DEBUGMODE=true
      - DB_HOST=postgresd
      - DB_PASSWORD=ThisIsUkamaXPass
      - DB_USER=postgres
      - QUEUE_URI=amqp://guest:guest@rabbitmq:5672
      - MSGCLIENT_HOST=msg-client-registry:9095
      - ORG_SERVICE_HOST=org
      - ORG_SERVICE_PORT=9090
      - UKAMA_UUID=${UKAMA_UUID}
      - ORGOWNERUUID=851e0abc-7e91-4206-8c85-498e16f91e67
      - 
```
But I guess we can do this on a separate issue

File: systems/registry/network/cmd/server/main.go (hunk 3 above)
Comment (delvatt) [reply systems/registry/network/cmd/server/main.go:54]: This is so weird!

File: systems/data-plan/package/pkg/db/package_repo.go [hunk 4]
```diff
@@ -54,11 +54,9 @@ func (p *packageRepo) GetByOrg(orgId uuid.UUID) ([]Package, error) {
 }
 
 func (r *packageRepo) Delete(uuid uuid.UUID) error {
-	result := r.Db.GetGormDb().Where("uuid = ?", uuid).Delete(&Package{})
-	if result.Error != nil {
```
[the remark this answers is not in the document]
Comment (salman-codewar) [reply systems/data-plan/package/pkg/db/package_repo.go:58]: addressed

File: systems/docker-compose.yml (hunk 1 above)
Comment (salman-codewar) [reply systems/docker-compose.yml:173]: addressed

File: systems/docker-compose.yml (hunk 2 above)
Comment (salman-codewar) [reply systems/docker-compose.yml]: Will address it in separate PR

**Pull request merged**